### ISSUE TITLE
Create exclusion parameters for jobs shortcode

### ISSUE THREAD

**Issue by cena** [Automattic/WP-Job-Manager#2078]
It is not currently possible to exclude jobs from the jobs listing, except by workarounds like putting all desired jobs in a category, and the desired excluded jobs in another,  then using [jobs  categories="x, y"], leaving out the excluded category.  

Allow exclusions based on job ID.

requested in 3552837-zen.

**Comment by tripflex** [comment]
Yeah this shouldn't be too difficult to do, just need to make sure that `post__not_in ` can be passed through shortcode to `get_job_listings` function

**Comment by Dominic-Marcelino** [comment]
Will this be taken into consideration? 

We use the shortcode to display other jobs from the category under the current one and would like to exclude the current one.
